api/rpc/server.go

```diff
@@ -44,7 +44,19 @@ func NewServer(address, port string, authDisabled bool, secret jwt.Signer) *Serv
 	}
 	srv.srv.Handler = &auth.Handler{
 		Verify: srv.verifyAuth,
-		Next:   rpc.ServeHTTP,
+		Next: func(w http.ResponseWriter, r *http.Request) {
+			if origin := r.Header.Get("Origin"); origin != "" {
+				w.Header().Set("Access-Control-Allow-Origin", origin)
```

Comment:
Any reason you dont set simply a `*`? AFAIU, it eill have the same effect

Reply:
Doesn't work. Was my first thought too, but it had something to do with the OPTIONS request IIRC. I'll post a link tomorrow when i'm back at the computer